crates/nu-parser/src/parser.rs

```diff
@@ -5537,6 +5550,7 @@ pub fn parse_expression(working_set: &mut StateWorkingSet, spans: &[Span]) -> Ex
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Not clear to me whether I need to keep `parse_variable` itself around, it's `pub` so I'm assuming it's part of _some_ public API but I've removed the only usage I saw in the repo.

Reply:
I think it's good to remove it